django/shortcuts.py

```diff
@@ -65,11 +73,11 @@ def _get_queryset(klass):
     """
     # If it is a model class or anything else with ._default_manager
     if hasattr(klass, '_default_manager'):
-        return klass._default_manager.all()
+        return klass._default_manager.all()  # type: ignore
     return klass
 
 
-def get_object_or_404(klass, *args, **kwargs):
+def get_object_or_404(klass: Union[Model, Manager, QuerySet], *args: Any, **kwargs: Any) -> Model:
```

Comment:
I think `klass` shouldn't be a `Model` (which mean a specific instance of a Model) but `Type[Model]` which means a Model class. Same applies to `get_list_or_404`